specification/cluster/reroute/ClusterRerouteResponse.ts

```diff
@@ -18,9 +18,10 @@
  */
 
 import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
+import { AcknowledgedResponseBase } from '@_types/Base'
 import { RerouteExplanation } from './types'
 
-export class Response {
+export class Response extends AcknowledgedResponseBase {
```

Comment:
Please don't extend a parent class if you are also defining the body, add the parameters from `AcknowledgedResponseBase.body` here.